File: C/Colleen.c
#include <stdio.h>

/* comment outsid main function */

void print(){
	char *str = "#include <stdio.h>%c%c/* comment outside main function */%c%cvoid print(){%c%cchar *str = %c%s%c;%c%cprintf(str, 10, 10, 10, 10, 10, 9, 34, str, 34, 10, 9, 10, 10, 10, 10, 9, 10, 9, 10);%c}%c%cint main() {%c%c/* comment inside main function */%c%cprint();%c}";
	printf(str, 10, 10, 10, 10, 10, 9, 34, str, 34, 10, 9, 10, 10, 10, 10, 9, 10, 9, 10);
}

int main() {
	/* comment inside main function */
	print();
}

File: Python/Colleen.py
# Comment outsid main function

def printFunction():
	str = '# Comment outsid main function\n\ndef printFunction():\n\tstr = {!r}\n\tprint(str.format(str))\n\ndef main():\n\t# Comment inside main function\n\tprintFunction()\n\nif __name__ == "__main__":\n\tmain()'
	print(str.format(str))

def main():
	# Comment inside main function
	printFunction()

if __name__ == "__main__":
	main()
